### ISSUE TITLE
Checkbox list - Need more space and remove list dot

### ISSUE THREAD

**Issue by ronindesign** [fulstech/markin#55]
I have two style suggestions (not bugs):
- Add more whitespace before / after checkboxes.
- Remove the prefixed black "list" dot before a checkbox in a list

When I use code for checkbox list, as in examples:
```
- [x] this is a complete item
- [ ] this is an incomplete item
```
This is what preview looks like in Jira's issue description:

![image](https://user-images.githubusercontent.com/1414401/42726218-99265e30-8745-11e8-9ff9-939f1063aaf4.png)

As you can see:
- (Almost) no space after the check box.
- Black list dots prefix to each checkbox is annoying / confusing.

Recommend:
- Remove the prefixed list dots.
- Add one space (or `&nbsp`) to both sides of checkbox. Or maybe 2 spaces to the left and 1 to the right since a user naturally adds 1 space after the `[ ]` when typing out.

This is how GitHub does the spacing, as you can see:

- [x] this is a complete item
- [ ] this is an incomplete item

And if you inspect their preview / HTML rendering we see:

```html
<ul class="contains-task-list">
  <li class="task-list-item">
    <input type="checkbox" id="" disabled="" class="task-list-item-checkbox" checked=""> this is a complete item
  </li>
  <li class="task-list-item">
    <input type="checkbox" id="" disabled="" class="task-list-item-checkbox"> this is an incomplete item
  </li>
</ul>
```

with CSS style as follows:

```css
.task-list-item-checkbox {
    margin: 0 0.2em 0.25em -1.6em;
    ...
}
```
As you can see, there is left and right offset.

**Comment by ptim** [comment]
I noticed this, too... 
workaround: I found I could lose the bullet, and increase the space, by typing:
`[ ]  item (not the double space before text)`

**Comment by ronindesign** [comment]
@ptim yeah, I tried that, but (if I remember correctly), this creates uneven spacing on the left side of checkbox... I'm picky I know!
